CHANGELOG.rst

```diff
@@ -2,6 +2,151 @@
 Release Notes
 =============
 
+.. _Release Notes_7.24.0:
+
+7.24.0 / 6.24.0
+======
+
+.. _Release Notes_7.24.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2020-12-01
```

Comment:
Should we change this to today's date? Thursday? Leave it and change it later?

Reply:
My plan was to change it tomorrow but let's actually write placeholder for now, less confusing if someone falls upon it